selfdrive/controls/controlsd.py

```diff
@@ -56,6 +57,9 @@ def __init__(self) -> None:
     elif self.CP.lateralTuning.which() == 'torque':
       self.LaC = LatControlTorque(self.CP, self.CI)
 
+    data_services = list(self.sm.data.keys()) + ['selfdriveStateSP']
```

Comment:
It kinda feels like we are injecting ourselves "late" within the "flow". Should perhaps we be earlier were the sm was instantiated?

Reply:
This happens at init, so it does not matter where this is handled as long as it's contained within init.

Reply:
I think it would be better if it happens alongside the other SM added for this